Dockerfile

```diff
@@ -19,18 +19,6 @@ RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
   && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
   && apt-get update && apt-get install yarn
 
-# PhantomJS is required for running tests
```

Comment:
Does chromedriver come pre-installed on the ruby image?

Reply:
Good question. I don't think it is. JS specs weren't able to run in Docker with Poltergeist/PhantomJS, so I don't think this should hold up this PR. I will keep trying to find a solution for Docker.